README.md

```diff
@@ -46,7 +59,17 @@ var Panda = function(name, age) {
 ```
 Your Answer:
 ```js
-// your code here
+
+var Panda = function(name, age) {
+  this.name = name;
+  this.age = age;
+  this.num_bamboo_eaten = 0;
+  this.eat_bamboo = function() {
+    this.num_bamboo_eaten++;
```

Comment:
In this case we would want to define `eat_bamboo` on the prototype instead of the constructor function so we would not be creating duplicate copies of this method each time a new instance is created


```
Panda.prototype.eat_bamboo = function(){
  this.num_bamboo_eaten += 1;
}
```